### ISSUE TITLE
[BUG] "Internal Error: Could not run Flutter command" when using fvm flutter

### ISSUE THREAD

**Issue by devkarim** [leoafarias/fvm#143]
**Describe the bug**
Getting ```Internal Error: Could not run Flutter command``` as an error message whenever I try to use ```fvm flutter``` however it works totally fine, I just don't know why that error message pops up every time I use the command.

**To Reproduce**
1. Open up PowerShell 7 or Windows PowerShell.
2. Try using ```fvm flutter```

**Expected behavior**
The command actually works as expected, I just don't know why that error message happens.

**Logs**
```

Internal Error: Could not run Flutter command
#0      runFlutterCmd (package:fvm/src/flutter_tools/flutter_tools.dart:43:5)
<asynchronous suspension>
#1      FlutterCommand.run (package:fvm/src/commands/flutter_command.dart:38:13)
<asynchronous suspension>
#2      CommandRunner.runCommand (package:args/command_runner.dart:197:27)
#3      CommandRunner.run.<anonymous closure> (package:args/command_runner.dart:112:25)
#4      new Future.sync (dart:async/future.dart:223:31)
#5      CommandRunner.run (package:args/command_runner.dart:112:14)
#6      fvmRunner (package:fvm/src/runner.dart:34:23)
#7      main (file:///C:/Users/Karim/AppData/Local/Pub/Cache/hosted/pub.dartlang.org/fvm-1.3.0-dev.0/bin/main.dart:4:9)
#8      _startIsolate.<anonymous closure> (dart:isolate-patch/isolate_patch.dart:299:32)
#9      _RawReceivePortImpl._handleMessage (dart:isolate-patch/isolate_patch.dart:168:12)

Unhandled exception:
Internal Error: Could not run Flutter command
#0      runFlutterCmd (package:fvm/src/flutter_tools/flutter_tools.dart:43:5)
<asynchronous suspension>
#1      FlutterCommand.run (package:fvm/src/commands/flutter_command.dart:38:13)
<asynchronous suspension>
#2      CommandRunner.runCommand (package:args/command_runner.dart:197:27)
#3      CommandRunner.run.<anonymous closure> (package:args/command_runner.dart:112:25)
#4      new Future.sync (dart:async/future.dart:223:31)
#5      CommandRunner.run (package:args/command_runner.dart:112:14)
#6      fvmRunner (package:fvm/src/runner.dart:34:23)
#7      main (file:///C:/Users/Karim/AppData/Local/Pub/Cache/hosted/pub.dartlang.org/fvm-1.3.0-dev.0/bin/main.dart:4:9)
#8      _startIsolate.<anonymous closure> (dart:isolate-patch/isolate_patch.dart:299:32)
#9      _RawReceivePortImpl._handleMessage (dart:isolate-patch/isolate_patch.dart:168:12)
```

**Desktop (please complete the following information):**
 - OS: Windows 10
 - FVM Version 1.3.0-dev.0
 - Tried Windows PowerShell and PowerShell 7.

By the way, is there a way where I can upgrade a specific channel without using it globally via ```fvm use <version> --global``` first? I have also tried the new GUI and it looks so neat. Just one issue I noticed is that I tried upgrading from there but it got stuck for some reason.

**Comment by leoafarias** [comment]
@devkarim thanks for reporting this. This happens when flutter returns an exception or exitCode is not 0. I will just return whatever error code flutter returns.

Regarding the upgrade I think this is what you are looking for https://github.com/leoafarias/fvm#upgrade-the-current-sdk-version

**Comment by devkarim** [comment]
> @devkarim thanks for reporting this. This happens when flutter returns an exception or exitCode is not 0. I will just return whatever error code flutter returns.
> 
> Regarding the upgrade I think this is what you are looking for https://github.com/leoafarias/fvm#upgrade-the-current-sdk-version

Glad to hear, it's harmless anyway as the command works as expected. Just wondered whether it breaks anything while having it around

And yes I saw that link that's why I asked, I just thought if I could upgrade a specific channel like ```fvm upgrade beta``` or something because now I think I have to use ```fvm use beta --global``` then ```fvm flutter upgrade``` am I right?

**Comment by leoafarias** [comment]
@devkarim  you don't have to use global depending on where you are at. If you want to open a new issue [enhancement]. I have the functionality completed because of the GUI. It allows you to upgrade, so I can hook in on the CLI also. 

Also I updated a new version 1.3.0-dev.1. If you could test it out to see if it fixes this problem it would be great.

**Comment by devkarim** [comment]
> @devkarim you don't have to use global depending on where you are at. If you want to open a new issue [enhancement]. I have the functionality completed because of the GUI. It allows you to upgrade, so I can hook in on the CLI also.
> 
> Also I updated a new version 1.3.0-dev.1. If you could test it out to see if it fixes this problem it would be great.

Yes, this is exactly what I mean. That's why I wanted to give the GUI a shot.

The new version outputs a different error message:
```
StdinException: Error setting terminal echo mode, OS Error: The parameter is incorrect.
, errno = 87
#0      Stdin.echoMode= (dart:io-patch/stdio_patch.dart:84:7)
#1      runFlutterCmd (package:fvm/src/flutter_tools/flutter_tools.dart:37:11)
<asynchronous suspension>
#2      FlutterCommand.run (package:fvm/src/commands/flutter_command.dart:38:13)
<asynchronous suspension>
#3      CommandRunner.runCommand (package:args/command_runner.dart:197:27)
#4      CommandRunner.run.<anonymous closure> (package:args/command_runner.dart:112:25)
#5      new Future.sync (dart:async/future.dart:223:31)
#6      CommandRunner.run (package:args/command_runner.dart:112:14)
#7      fvmRunner (package:fvm/src/runner.dart:34:23)
#8      main (file:///C:/Users/Karim/AppData/Local/Pub/Cache/hosted/pub.dartlang.org/fvm-1.3.0-dev.1/bin/main.dart:4:9)
#9      _startIsolate.<anonymous closure> (dart:isolate-patch/isolate_patch.dart:299:32)
#10     _RawReceivePortImpl._handleMessage (dart:isolate-patch/isolate_patch.dart:168:12)

Unhandled exception:
StdinException: Error setting terminal echo mode, OS Error: The parameter is incorrect.
, errno = 87
#0      Stdin.echoMode= (dart:io-patch/stdio_patch.dart:84:7)
#1      runFlutterCmd (package:fvm/src/flutter_tools/flutter_tools.dart:37:11)
<asynchronous suspension>
#2      FlutterCommand.run (package:fvm/src/commands/flutter_command.dart:38:13)
<asynchronous suspension>
#3      CommandRunner.runCommand (package:args/command_runner.dart:197:27)
#4      CommandRunner.run.<anonymous closure> (package:args/command_runner.dart:112:25)
#5      new Future.sync (dart:async/future.dart:223:31)
#6      CommandRunner.run (package:args/command_runner.dart:112:14)
#7      fvmRunner (package:fvm/src/runner.dart:34:23)
#8      main (file:///C:/Users/Karim/AppData/Local/Pub/Cache/hosted/pub.dartlang.org/fvm-1.3.0-dev.1/bin/main.dart:4:9)
#9      _startIsolate.<anonymous closure> (dart:isolate-patch/isolate_patch.dart:299:32)
#10     _RawReceivePortImpl._handleMessage (dart:isolate-patch/isolate_patch.dart:168:12)```

**Comment by leoafarias** [comment]
Fixed on version `1.3.0-dev.2`
